### PR TITLE
make function declaration consistent

### PR DESCRIPTION
**Pull request by denar90**
### Proposed changes
 - in some places we have `foo: function() {...}` in some `bar() {...}`, so I proposing to make it consistent in format `bar() {...}`

Link to the issue:
I think It's a bit related to https://github.com/marionettejs/backbone.marionette/issues/2858
So for Classes we can have `bar() {...}` declaration and for utils 
```
function foo() {
  ...
}

function bar() {
  ...
}
export {foo, bar}
```
as it is now.

### CODE REVIEW

**Comment by paulfalgout** [comment]
Works for me.  We could enforce this with the method option of the eslint rule: http://eslint.org/docs/rules/object-shorthand#options

**Comment by JSteunou** [comment]
+1 on the eslint rule

File: .eslintrc [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "strict": 0,
     "valid-jsdoc": 2,
     "valid-typeof": 2,
-    "wrap-iife": [ 2, "inside" ]
+    "wrap-iife": [ 2, "inside" ],
+    "object-shorthand": [ 2, "methods" ]
```
Comment (paulfalgout) [inline .eslintrc]: sorry to be picky, but can you add this alphabetically?

File: .eslintrc [hunk 2]
```diff
@@ -54,7 +54,7 @@
     "no-undef": 2,
     "no-unused-vars": [ 1, { "args": "none" } ],
     "no-with": 2,
-    "one-var": [ 2, "never" ],
+    "no-var": 2,
```
Comment (paulfalgout) [inline .eslintrc]: same here, can you move it up one?

File: .eslintrc (hunk 1 above)
Comment (denar90) [inline .eslintrc]: Sure. Fair enough. It was me who didn't notice that they are sorted alphabetically.

**Comment by paulfalgout** [comment]
👍

**Comment by JSteunou** [comment]
:+1:

**Pull request merged**